docs/swagger_v3/names.spec.yaml

```diff
@@ -219,16 +209,19 @@ schemas:
       source_tx_hash:
         $ref: '#/components/schemas/TransactionHash'
       source_tx_type:
-        type: string
+        enum:
```

Comment:
it looks like `type: string` should be present aside `enum:`
https://swagger.io/docs/specification/v3_0/data-models/enums/